tests/php/sync/test_class.jetpack-sync-posts.php

```diff
@@ -672,6 +672,22 @@ function test_embed_is_disabled_on_the_content_filter_during_sync() {
 		}
 	}
 
+	function test_do_not_sync_non_public_post_types_filtered_post_content() {
+		$args = array(
+			'public' => false,
+			'label'  => 'Non Public'
+		);
+		register_post_type( 'non_public', $args );
+
+		$post_id = $this->factory->post->create( array( 'post_type' => 'non_public' ) );
+
+		$this->sender->do_sync();
+		$synced_post = $this->server_replica_storage->get_post( $post_id );
+
+		$this->assertEquals( '', $synced_post->post_content_filtered );
+		$this->assertEquals( '', $synced_post->post_excerpt_filtered );
```

Comment:
Nope one is for post_**content**_filterd and the other for post_**excerpt**_filterd